File: src/SamplePublishing/RA.SamplesForDocumentation/SampleModels/Credential.cs
using System.Collections.Generic;

namespace RA.SamplesForDocumentation.SampleModels
{
	public class Credential : BaseModel
	{
        //provide valid concept from schema 
        public string CredentialType { get; set; }
		public string Type
		{
			get { return CredentialType; }
		}
		//provide credential status type, with a default of Active
        public string CredentialStatusType { get; set; } = "Active";
        public string InLanguage { get; set; }
        public string AvailabilityListing { get; set; }
		public string AvailableOnlineAt { get; set; }
		public string ImageUrl { get; set; }
		public string LatestVersionUrl { get; set; }
		public string PreviousVersion { get; set; }
		public List<string> Subject { get; set; }
		public List<string> Keyword { get; set; }
        public List<string> OccupationCodes { get; set; }
		public Organization OwningOrganization { get; set; }
    }
}

File: src/SamplePublishing/VisualBasicPublishingSamples/SampleModels/Credential.vb
Public Class Credential
	Inherits BaseModel

	Public Property CredentialType As String
	Public Property AvailabilityListing As String
	Public Property AvailableOnlineAt As String
	Public Property CodedNotation As String
	Public Property ImageUrl As String
	Public Property LatestVersionUrl As String
	Public Property PreviousVersion As String
	Public Property Subject As List(Of String)
	Public Property Keyword As List(Of String)
End Class

